src/js/components/Notification/Notification.js

```diff
@@ -0,0 +1,120 @@
+import React, { useContext } from 'react';
+import { ThemeContext } from 'styled-components';
+import { defaultProps } from '../../default-props';
+
+import { Layer } from '../Layer';
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { Text } from '../Text';
+
+const Notification = ({ toast, message, body, status, onClose }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+
+  const {
+    gap: containerGap,
+    elevation: containerElevation,
+    round: containerRound,
+  } = theme.notification.container;
+
+  const { pad: iconContainerPad, round: iconContainerRound } =
+    theme.notification.iconContainer;
+
+  const { pad: textContainerPad } = theme.notification.textContainer;
+
+  const { size: messageTextSize, weight: messageTextWeight } =
+    theme.notification.messageText;
+
+  const { size: bodyTextSize } = theme.notification.bodyText;
+
+  const { margin: buttonMargin } = theme.notification.button;
+
+  const Icons = {
+    StatusGood: theme.notification.icons.statusGood,
+    StatusWarning: theme.notification.icons.statusWarning,
+    StatusUnknown: theme.notification.icons.statusUnknown,
+    FormClose: theme.notification.icons.closeButton,
+  };
+
+  const iconColor = theme.notification.icons.color;
+
+  let icon;
+  let color;
+  switch (status) {
+    case 'warning':
+      color = theme.notification.status.warning.color;
```

Comment:
It would be more consistent with other parts of the theme to just have `theme.notification.warning`.